WasatchNET/EEPROM.cs

```diff
@@ -1973,6 +2024,21 @@ protected bool writeParse()
                         }
                         pixel++;
                     }
+                    int start_index;
+                    foreach(string libName in lib_names)
+                    {
+                        if(names_written < 4)

```

Comment:
since this is copy-pasted code from above, recommend adding a function "int libraryNamePage(int index)" so the multiple implementations don't drift out of sync.